langs/racket/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM alpine:3.20 AS builder
+
+RUN apk add --no-cache build-base curl git
+
+RUN curl -L https://github.com/racket/racket/archive/refs/tags/v8.14.tar.gz \
+  | tar xz --strip-components 1
+
+RUN make -j`nproc` base
```

Comment:
> #### 1.7.1. Building Minimal Racket
> 
> Instead of using the top-level makefile, you can go into `"racket/src"` and follow the `"README.txt"` there, which gives you more configuration options.
> 
> If you don’t want any special configuration and you just want the base build, you can use `make base` with the top-level makefile.

That explains my choice for `make base` because the minimal base build only includes `collects`, the standard library.

> ### 1.6. More Instructions: Building Racket CS and Racket BC
> 
> The default build of Racket, also known as Racket CS, uses and incorporates Chez Scheme. Chez Scheme sources are included in the Racket repository.

Sorry for interrupting, but that's kind of messed up. Wouldn't that require like some drastic reset measure? It'd be either something along those lines, or dead Racket's opportunities. Now, please proceed.

> The `make bc` target (or `make bc-as-is` for a rebuild) builds an older variant of Racket, called Racket BC, which does not use Chez Scheme.

Now, that's better, so I've something coming in momentarily.